### DISCUSSION TITLE
When compiling the program, it does not find the smartwatch.

### DISCUSSION THREAD

**Discussion by ArtemIk1515** [zepp-health/zeppos-forum#290] [Q&A] [answered]
When using the "zeus dev" or "zeus preview" command, it is difficult to select the device for which the program is compiled. There are three options to choose from: Balance, Kunlun, KunlunWA (the last 2 - I don't even know anything about them). At the same time, I have downloaded only GTS 4 mini in the simulator. Where does the compiler get the rest and versions from? And how do I choose mine?
Thank you in advance!

**Comment by SashaCX75** [comment, marked as answer]
The compiler takes data from the app.json file. This file is formed when creating a programme.
Judging by the results you have created a programme for ZeppOS 3, but for GTS 4 mini you need to create a programme for ZeppOS 1.
